src/components/ThreeDotsMenu/index.tsx

```diff
@@ -67,12 +76,14 @@ function ThreeDotsMenu({
     shouldOverlay = false,
     shouldSetModalVisibility = true,
     disabled = false,
+    modal = {},
 }: ThreeDotsMenuProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const [isPopupMenuVisible, setPopupMenuVisible] = useState(false);
     const buttonRef = useRef(null);
     const {translate} = useLocalize();
```

Comment:
NIT: I think if `modal?.willAlertModalBecomeVisible` is `true`, the next ? after modal is not necessary.

Reply:
I meant it will reach to checking `!modal?.isPopover` only when `modal?.willAlertModalBecomeVisible` is `true` which guarantees that `modal` is not `null` so in the `modal?.isPopover` being evaluated only after `modal?.willAlertModalBecomeVisible` is `true`, modal check for null in `modal?.isPopover` does not appear to be required.

It is a non issue and a very minor nitpick anyway 😁